### ISSUE TITLE
CSS in the repo

### ISSUE THREAD

**Issue by makanikki** [AurousGroup/frontend#16]
Hi

Here is a little something: Why do you have the compiled style in the repo? 
As we already have scss files, we only need to compile after pulling
I think it's also quite annoying to have to push the css file with the scss one when we make a pull request

**Comment by dmxt** [comment]
Because the software read CSS file, not SCSS so it has to be compiled.

**Comment by makanikki** [comment]
Yep no problem with this, but why do we have compiled filed in the repo. Can't we compile them during deployment process?

**Comment by dmxt** [comment]
With compiled CSS, any users can view the frontend without using a compiler, and JavaScript developers can contribute this repo with JavaScript without needing a SCSS compiler because they don't touch SCSS/CSS. 

There's also nothing wrong with compiling pre-exising CSS file when contributes change something in SCSS, also with pre-existing CSS file, they would know what CSS file (naming, structure) to compile.

**Comment by carloscheddar** [comment]
I agree that having the compiled `CSS` in the repo is very convenient but, it opens up some issues.

If you have multiple users committing compiled `CSS` alongside their `SCSS` changes then you will have merge conflicts frequently on the compiled `CSS` even if the `SCSS` files are completely unrelated. 

Maybe a good compromise will be to only commit the compiled `CSS` in a release branch once all the style changes are merged.

**Comment by makanikki** [comment]
I didn't even think about conflicts that will obviously happen if many user work on the same `SCSS` file.
What I've used to do in many project with preprocessed style is to only commit `SCSS`/`LESS`  files
If someone want to work on the style, he compiles it on the way he want, respecting specific rules like the target file name for instance.

Those who need to work on the style without modifying it can read `SCSS`/`LESS` files (I think it's not so difficult to understand, even for those who didn't ever work with those, or it can be the opportunity to learn new things, nothing bad there)

Like this we exclude conflicts on final files, and we don't have to push 2 files instead of 1 when modifications are done.

Pushing only compiled files will be difficult to me. It means if we want to work on style with a preprocessor, we need to convert `CSS` to `SCSS`/`LESS`, work on it and then reconvert it to `CSS`

If rules are defined, everyone work with it the same way and it's not ambiguous and complicated anymore 

(Thanks for reading me, got a little bit inspired !)

**Comment by glauberramos** [comment]
+ 1 to this, there is no need for commiting the compiled css files. It is not necessary since you will not change that file directly, the repo should only contain code developers will change.

**Comment by makanikki** [comment]
I agree with @glauberramos

**Comment by dmxt** [comment]
What about we leave .CSS in the repo for developers or previewer to use, and make a guideline that we don't allow changes to main.css on pull request?
